core/eth/tx.go

```diff
@@ -603,6 +604,38 @@ func (t *Transactor) GetQuorumCount(ctx context.Context, blockNumber uint32) (ui
 	})
 }
 
+func (t *Transactor) GetQuorumSecurityParams(ctx context.Context) ([]*core.SecurityParam, error) {
+	adversaryThresholdPercentegesBytes, err := t.Bindings.EigenDAServiceManager.QuorumAdversaryThresholdPercentages(&bind.CallOpts{
```

Comment:
Does the contracts submodule need to be updated as part of this PR?

Reply:
no they should not need to be

Reply:
Probably not the submodule, but the contract changes to `EigenDAServiceManager` need to be part of this PR, no?

Reply:
I've rebased to include the changes.